ec2/sc-ec2-linux-jumpcloud.yaml

```diff
@@ -355,6 +349,45 @@ Resources:
             -
               yum:
                 jq: []
+          files:
+            /opt/sage/bin/make_env_vars_file.sh:
+              content: !Sub |
+                #! /bin/bash
+
+                EC2_INSTANCE_ID=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/instance-id)
+                ROOT_DISK_ID=$(/usr/bin/aws ec2 describe-volumes \
+                  --region $AWS_REGION \
+                  --filters Name=attachment.instance-id,Values=$EC2_INSTANCE_ID \
+                  --query Volumes[].VolumeId --out text)
+                EC2_INSTANCE_TAGS=$(aws --region $AWS_REGION \
+                  ec2 describe-tags \
+                  --filters Name=resource-id,Values=$EC2_INSTANCE_ID)
+                extract_tag_value() {
+                  echo $EC2_INSTANCE_TAGS | jq -j --arg KEYNAME "$1" '.Tags[] | select(.Key == $KEYNAME).Value '
+                }
+                DEPARTMENT=$(extract_tag_value Department)
+                PROJECT=$(extract_tag_value Project)
+                OWNER_EMAIL=$(extract_tag_value OwnerEmail)
+
+                echo "export AWS_REGION=$AWS_REGION" >> /opt/sage/bin/export_env_vars.sh
+                echo "export STACK_NAME=$STACK_NAME" >> /opt/sage/bin/export_env_vars.sh
+                echo "export EC2_INSTANCE_ID=$EC2_INSTANCE_ID" >> /opt/sage/bin/export_env_vars.sh
+                echo "export ROOT_DISK_ID='$ROOT_DISK_ID'" >> /opt/sage/bin/export_env_vars.sh
+                echo "export DEPARTMENT=$DEPARTMENT" >> /opt/sage/bin/export_env_vars.sh
+                echo "export PROJECT=$PROJECT" >> /opt/sage/bin/export_env_vars.sh
+                echo "export OWNER_EMAIL=$OWNER_EMAIL" >> /opt/sage/bin/export_env_vars.sh
+                chmod +x /opt/sage/bin/export_env_vars.sh
+
+              mode: "00644"
```

Comment:
should this be `00744` instead, that would make it executable?

Reply:
Oddly this is working as is. I'll change the mode.